target/linux/lantiq/files-4.19/arch/mips/boot/dts/lantiq/grx390.dtsi

```diff
@@ -0,0 +1,322 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+#include <dt-bindings/gpio/gpio.h>
+
```

Comment:
No need to add this for kernel 4.19 any more, it should be sufficient if this is only working with kernel 5.4